lib/SPIRV/SPIRVReader.cpp

```diff
@@ -835,6 +835,9 @@ void SPIRVToLLVM::setLLVMLoopMetadata(const LoopInstType *LM,
           continue;
 
         Value *AccessedPointer = GEP->getPointerOperand();
+        if (auto *LI = dyn_cast<LoadInst>(AccessedPointer))
+          if (Value *LoadOp = LI->getPointerOperand())
```

Comment:
Do we really need this as an `if` statement? I mean is it expected to encounter a `load` instruction without its pointer operand?